api/v1alpha1/clusterextension_types.go

```diff
@@ -204,7 +276,7 @@ type ClusterExtensionSpec struct {
 	// the bundle selection process.
 	//
 	//+optional
-	CatalogSelector metav1.LabelSelector `json:"catalogSelector,omitempty"`
+	LabelSelector metav1.LabelSelector `json:"catalogSelector,omitempty"`
```

Comment:
We should also make sure we update:
- The JSON field name
- The comment that references the JSON field name

Reply:
Also a question I think worth having some discussion on since we can't change it until a v2 of the API - is `labelSelector` to verbose? is something like `selector` or `labels` sufficiently descriptive?

Reply:
Shortened this to `Selector` to match k8s Deployments (I incorrectly remembered this field name as `LabelSelector`). I was only doing this for consistency w/K8s, but turns out I was inconsistent...